aggregator/src/aggregator/collection_job_driver.rs

```diff
@@ -268,17 +267,16 @@ where
                     // There is no pre-existing finished collection job, but the collection job is
                     // ready to be completed. Read batch aggregations so that we can compute the
                     // final aggregate value.
-                    let mut batch_aggregations =
-                        B::get_batch_aggregations_for_collection_identifier(
-                            tx,
-                            lease.leased().task_id(),
-                            lease.leased().time_precision(),
-                            vdaf.as_ref(),
-                            &collection_identifier,
-                            &aggregation_param,
-                        )
-                        .await?;
-
+                    let batch_aggregations = B::get_batch_aggregations_for_collection_identifier(
+                        tx,
+                        lease.leased().task_id(),
+                        lease.leased().time_precision(),
+                        vdaf.as_ref(),
+                        &collection_identifier,
+                        &aggregation_param,
+                    )
+                    .await?
+                    .into_iter()
```

Comment:
We could split this into two statements to avoid formatting strangeness from comments between chained method calls.
```suggestion
                    .await?;
```